spec/spec_helper.rb

```diff
@@ -98,3 +98,8 @@
   # as the one that triggered the failure.
   Kernel.srand config.seed
 end
+
+require 'retriable'
+Retriable.configure do |c|
+  c.tries = 2
+end
```

Comment:
**suggestion:** Move this to `spec/support`. Uncomment this line https://github.com/DigitalNZ/supplejack_harvester/blob/b80146ff56083d3642b1f6faca40be6f983a6f4b/spec/rails_helper.rb#L28 and remove this one. https://github.com/DigitalNZ/supplejack_harvester/blob/b80146ff56083d3642b1f6faca40be6f983a6f4b/spec/rails_helper.rb#L11